CHANGELOG.md

```diff
@@ -215,6 +215,7 @@ are doing mtv adjustments during precollision.
   * `EventEmitter`s
   * `GraphicsSystem` entity iteration
   * `PointerSystem` entity iteration
+- Perf improvements to `GraphicsGroup` by reducing per draw allocations in bounds calculations
```

Comment:
This is the only change we should keep on the changelog, we should revert the rest